dependencies.yml

```diff
@@ -417,7 +417,6 @@ org.jctools:
 
 org.jetbrains.kotlin:
   kotlin-allopen: { version: &KOTLIN_VERSION '1.3.61' }
```

Comment:
Could you add the same comment here? So that a dev does not forget to update `build.gradle`?

Reply:
Good Idea!